### ISSUE TITLE
RaspiCam_Still Locks up after capturing

### ISSUE THREAD

**Issue by PreciousRoy0** [cedricve/raspicam#40]
Hi,

I am having trouble to get the raspicam_still_test working. compiling and linking went without any problems. running raspicam_test works just fine but when I want to run raspicam_still_test it locks up and stops seeming to wait for something. I need the raspicam_still_test because I want to take some detailed pictures.

    pi@rootcampi1:~/Downloads/raspicam/build/utils $ sudo raspicam_still_test 
    -w val : sets image width (2592 default)
    -h val : sets image height (1944 default)
    -iso val: set iso [100,800] (400 default)
    Initializing ...2592x1944
    capture

**Comment by dongdixidi** [comment]
hi,have you resloved it?
this problem seems to be hanppen when use raspicam v2

**Comment by muellerkyle** [comment]
I'm also running up against this on a Pi Zero W.

It hangs at the sem_wait call

**Comment by gallex** [comment]
on Pi 3B+ too

**Comment by PreciousRoy0** [comment]
Yes on the pi zero w and pi3b+. On the zero i use the 5mp cam and pi 3b cam v2

**Comment by akmal-bakar** [comment]
facing this issue too
